### PR TITLE
Update team page

### PR DESCRIPTION
**Pull request by cjdenio**
This PR updates the Team page to reflect the folks who are now at HQ (and who aren't anymore).

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/team**](https://site-lddqeaeys.hackclub.dev/team) |
|:-:|
|<a href="https://site-lddqeaeys.hackclub.dev/team">Screenshot of /team</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/td0jfbYEpQ_wHHcilFX_V82dFpfukN6b3TuYBzcnFvij1UNzJZns6-1E6F7-t2MGiwwjfD73dLeG_M1XnkazgA.png">(view full size)</a> |







Commit 83c452d5b4d6f72a2deb3755307b425a6a18056c (https://site-lddqeaeys.hackclub.dev).

**Review by sampoder — CHANGES_REQUESTED** [review]
Looks good to me! Should we move Jessica to acknowledgements? And make Ishan's bio past-tense?

File: pages/team.js [hunk 1]
```diff
@@ -309,6 +295,20 @@ and participating in academic competitions with his friends.`}
...
+                name="Ishan Goel"
+                teamRole="2022 Summer Communications Intern"
+                text="Ishan was a summer intern from Seattle! During the summer of 2022, he worked on shipping projects with partners to get the word out about Hack Club, and bring more people into the community."
+                img="https://github.com/quackduck.png"
+                pronouns="he/him"
+              />
+              <Bio
+                name="Jessica Card"
+                teamRole="Education Engineer"
+                text="Jessica is a self taught programmer originally from Alaska. She worked for over a decade as a software engineer at startups like GitHub and Bugsnag in San Francisco. She then left the web development world to learn how to make video games."
```
Comment (sampoder) [inline pages/team.js]:
```suggestion
                text="Jessica is a self taught programmer originally from Alaska. She worked for over a decade as a software engineer at startups like GitHub and Bugsnag in San Francisco. She then left the web development world to learn how to make video games. At Hack Club, Jessica brought her creative energy to an array of projects! Most notably, when she learnt Assembly along with Hack Clubbers to produce Some Assembly Required."
```

**Review by sampoder — APPROVED** [review]
Nice!

**Review by sampoder — APPROVED** [review]
Nice!

**Comment by sampoder** [comment]
(ah, I forgot... I had already reviewed... stupid me)

**Pull request merged**